src/services/permissions.js

```diff
@@ -92,7 +92,15 @@ module.exports = class PermissionsHelper {
 				})
 			}
 		} catch (error) {
-			throw error
+			if (error.name === common.SEQUELIZE_DATABASE_ERROR && error.parent?.code === '22003') {
```

Comment:
@priyanka-TL can we not check whether its integer or not in validator only?

Reply:
done